lib/coherence/controllers/invitation_controller.ex

```diff
@@ -79,7 +85,8 @@ defmodule Coherence.InvitationController do
                 add_error(changeset, :email,
                   Messages.backend().invitation_already_sent())}
           end
-        render(conn, "new.html", changeset: changeset)
+        conn
```

Comment:
Minor comment, but I don't believe this conforms to style guide. Single pipe here should be replaced with a direct call. i.e. `responds_with(conn, :invitation_create_error, %{changeset: changeset})`

The same comment applies to other areas in this PR.